schemas/configuration/v4/rule_engine.json.in

```diff
@@ -8,7 +8,8 @@
         "plugin_specific_configuration": {
             "type": "object",
             "properties": {}
-        }
+        },
+        "shared_memory_instance": {"type": "string"}
```

Comment:
We need to investigate whether this property is unique to the NREP (i.e. iRODS Rule Language REP / Native Rule Engine Plugin).

Reply:
You are right. After looking, this property does not exist for other rule engines.  Would we want to keep properties that do not always show up but not make it required so that if they do they still are type checked or do we want to only have the properties that are guaranteed to show up?

Reply:
We agreed that leaving `shared_memory_instance` as a property but not as a required property is fine.